### PR TITLE
[TTS]add multi-band melgan finetune scripts

### PR DESCRIPTION
**Pull request by yt605155624**
1. move StyleFastSpeech2Inference from fs2_style.py to fastspeech2.py for reuse
2. The finetune model is training now

### CODE REVIEW

File: examples/csmsc/voc3/finetune.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+fi
+
+if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
+  python3 local/link_wav.py \
+    --old-dump-dir=dump \
+    --dump-dir=dump_finetune 
+
+fi
+
+if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
```
Comment (zh794390558) [inline examples/csmsc/voc3/finetune.sh:28]: fintue不需要重新计算stats

File: paddlespeech/t2s/exps/fastspeech2/gen_gt_duration_mel.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```
Comment (zh794390558) [inline paddlespeech/t2s/exps/fastspeech2/gen_gt_duration_mel.py]: 文件名改为gen_gta_mel.py

File: examples/csmsc/voc3/finetune.sh (hunk 1 above)
Comment (yt605155624) [reply examples/csmsc/voc3/finetune.sh:28]: done

File: paddlespeech/t2s/exps/fastspeech2/gen_gt_duration_mel.py (hunk 2 above)
Comment (yt605155624) [reply paddlespeech/t2s/exps/fastspeech2/gen_gt_duration_mel.py]: done

**Review by zh794390558 — APPROVED** [review]
LGTM

**Pull request merged**